react/components/ProductAvailabilityWrapper.tsx

```diff
@@ -131,6 +154,11 @@ ProductAvailabilityWrapper.schema = {
       description: messages.highStockMessageDescription.id,
       type: 'string',
     },
+    showAvailabilityMessage: {
```

Comment:
Make sure to add the `showAvailabilty` property to the schema too. That way, users can enable this feature through site-editor entirely.